lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
Another argument in @anntzer's favor is that the scalar case is explicitly dealt with in the sentence above the parameters section.